datalad/support/gitrepo.py

```diff
@@ -2362,6 +2364,62 @@ def gc(self, allow_background=False, auto=False):
             cmd_options += ['--auto']
         self._git_custom_command('', cmd_options)
 
+    def _parse_gitmodules(self):
```

Comment:
The changes from this commit are almost purely code movement and are probably easiest to review with something like `git show --color-moved=dimmed_zebra --color-moved-ws=allow-indentation-change 1256169f`.